commands/cfe.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const Validation = require('../utils/validator');
+const myValidator = new Validation();
+const sendMessage = require('../utils/sendTableMessage');
+const Scrapper = require('../webScraping/cfeScrapper');
+
+/**
+ * Main function of the command
+ * @param { Message } message Channel to send the data
+ * @arg { searchItems } args Searching parameters
+ * @return { void }
+ */
+async function execute(message, args) {
+  if (!args || args.length == 0) {
+    return message.channel.send('The command needs a searching parameter.');
+  }
+  const route = `./assets/cfe_${args.join('').toLowerCase()}.json`;
+  args = args.join(' ');
+  const executeScrapper = myValidator.isFileLastUpdateIn(route);
+  if (!executeScrapper) {
+    message.reply('Is needed execute the scrapper, executing...');
+
+    try {
+      const cfeScrapper = new Scrapper(args);
+      // Show how many data has been obtained
+      cfeScrapper.printPercentage = (percentage) => {
+        message.reply(`Loading data ${percentage.toString()} %`);
+      };
```

Comment:
is there a reason why we override `printPercentage` here?

Reply:
So send the message to discord server, right now is a longer